client/app/adapters/application.js

```diff
@@ -2,5 +2,8 @@ import DS from 'ember-data';
 
 export default DS.JSONAPIAdapter.extend({
     host: 'http://127.0.0.1:8000',
-    namespace: 'api'
+    namespace: 'api',
+    headers :{
+        'Authorization' : 'Basic ASDASDASDASDASD'
```

Comment:
We can do something like this for settings that are not supposed to be committed (define it in environment.js and set it as an environment variable locally): https://github.com/CenterForOpenScience/isp/blob/develop/config/environment.js#L55